policy/release/tasks_test.rego

```diff
@@ -3,6 +3,29 @@ package policy.release.tasks
 import data.lib
 import data.lib.bundles
 
+test_all_tasks_present {
+	lib.assert_empty(deny) with data["task-bundles"] as bundles.bundle_data
+		with input.attestations as [{"predicate": {
```

Comment:
I think `input.attestation` can be generated via `_attestations_with_tasks(all_required_tasks)` that way we don’t need to maintain a second list of required tasks here

Reply:
Yes!